### PR TITLE
Changed 5.6, 7.0 and 7.1 images to be stretch based ( not jessie )

### PR DESCRIPTION
**Pull request by vidarl**

### CODE REVIEW

**Review by andrerom — CHANGES_REQUESTED** [review]
Should bump format version in .travis.yml and document it.

And for failures diff against the 7.2 image to add missing libs and make sure they are in sync

**Comment by vidarl** [comment]
> Should bump format version in .travis.yml and document it.

I see the variable `FORMAT_VERSION`, but it ain't use much... As far as I can see, the v1 is set as `$2` to `bin/.travis/push.sh` which you invoke manually when you want to upload images on dockerhub?

File: bin/.travis/test.sh [hunk 1]
```diff
@@ -33,6 +33,11 @@ if [ "$EZ_VERSION" = "" ]; then
     EZ_VERSION="^2.4"
 fi
 
+if [ "$EZ_VERSION" = "^2.4" ]; then
+    PHP_VARIANT='latest-node'
+else
+    PHP_VARIANT='latest'
+fi
```
Comment (andrerom) [inline bin/.travis/test.sh]: why do this? it will break as soon as we set ^2.5 or something else, why not just use latests-node further down?

**Comment by andrerom** [comment]
It's used here: https://github.com/ezsystems/docker-php/search?q=FORMAT_VERSION&unscoped_q=FORMAT_VERSION

It's part of the tag name, and with bump in OS, it should be bumped as well.

**Comment by vidarl** [comment]
ahh... For whatever reason I missed the use of FORMAT_VERSION in the after_success hook when I looked for it

**Comment by vidarl** [comment]
So, have now made `v2` for 5.6, 7.0 and 7.1 which is  based on stretch (same as 7.2 and 7.3). I kept building v1 of those images too for BC, only to discover that PHP does not update their 5.6 and 7.0 images any more. As a result of that, 5.6 and 7.0 on jessie won't build any more due to obsolete apt sources in the respective base images...

So, since `v1` for 5.6 and 7.0 anyway won't build currently, should we anyway base them on stretch without bumping the version? If so, then we should do the same with 7.1 IMO

**Comment by vidarl** [comment]
So, since jessie builds are broken now anyway, we'll simply change all v1 images to be stretch based

File: bin/.travis/test.sh (hunk 1 above)
Comment (andrerom) [reply bin/.travis/test.sh]: @vidarl remark on dropping this still stands

**Review by andrerom — APPROVED** [review]
+1 beside the remark on unneeded (?) complexity on latest-node logic, which will be prone to fail later.

File: bin/.travis/test.sh (hunk 1 above)
Comment (vidarl) [reply bin/.travis/test.sh]: Fixed

**Review by mnocon — COMMENTED** [review]
Looks good to me, I'd like to clarify just one thing:
@andrerom mentioned:
> Should bump format version in .travis.yml and document it.

Yet I don't see that change - will it happen after this is merged/in other way?

Still discovering how things work here exactly 😉

**Comment by vidarl** [comment]
@mnocon : The answer to your question regarding bumping version:
> So, since jessie builds are broken now anyway, we'll simply change all v1 images to be stretch based

**Comment by andrerom** [comment]
> Yet I don't see that change - will it happen after this is merged/in other way?

see the hidden comments, there was a back and forward until Vidar mentioned we should look at this as a bug given jessie images are broken. So we bump all, and you won't need to update docker image strings in other repos as a bonus.

**Review by mnocon — APPROVED** [review]
Thanks, all clear!

**Pull request merged**